### ISSUE TITLE
Is it possible to include/exclude a window from being managed via hotkey?

### ISSUE THREAD

**Issue by janus0** [fuhsjr00/bug.n#80]
It's fairly annoying having to go into the config file and add a new line of code any time a window is misbehaving. Since the code is pretty much fill in the blank with the window information, and there's a script to copy paste window info, could it some how be made so that the info gets copy/pasted into the correct fields inside the config file when a hotkey is pressed, and then a prompt to reload bug.n?

**Comment by joten** [comment]
Yes, it would be possible. But it may be error-prone since window titles and classes are often either to specific to match all related windows or they contain characters, which may prevent the regex to match; therefor manual intervention is necessary in those cases anyway.

-- You must use a lot of different applications
